### PR TITLE
Added initial files

### PR DESCRIPTION
**Pull request by DARSHAN9822**
"Add database schema with tables for customers, products, orders, order items,  categories, shipments and reviews with necessary relationships and constraints."

### CODE REVIEW

File: SRC/setup.sql [hunk 1]
```diff
@@ -0,0 +1,86 @@
+CREATE TABLE Customers (
```
Comment (eccentric93) [inline SRC/setup.sql]: CREATE TABLE if not exists

File: SRC/setup.sql [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+         customer_id INT PRIMARY KEY AUTO_INCREMENT,
+         name VARCHAR(100) NOT NULL,
+         email VARCHAR(100) NOT NULL UNIQUE,
+         phone VARCHAR(20) NOT NULL,
+         address TEXT
+    );
+
+
+CREATE TABLE Products (
```
Comment (eccentric93) [inline SRC/setup.sql]: Add comments on every table on what it is meant for

**Comment by eccentric93** [comment]
Add a new .sql file to insert dummy data into DB

**Comment by eccentric93** [comment]
Add a new .sql file to insert dummy data into DB

**Comment by eccentric93** [comment]
Update description of the pull request

File: SRC/setup.sql [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+        payment_date DATETIME DEFAULT CURRENT_TIMESTAMP,
+        amount DECIMAL(10, 2) NOT NULL,
+        status VARCHAR(20) NOT NULL,
+        FOREIGN KEY (order_id) REFERENCES Orders(order_id)
+    );
+
+CREATE TABLE Categories (
+         category_id INT PRIMARY KEY AUTO_INCREMENT,
+         name VARCHAR(100) NOT NULL ,
+         parent_category_id INT NOT NULL ,
```
Comment (eccentric93) [inline SRC/setup.sql]: This should be a NULL field

**Comment by DARSHAN9822** [comment]
done

**Comment by DARSHAN9822** [comment]
Done ....

**Comment by DARSHAN9822** [comment]
Please check and let me know if anything need to be change

**Review by eccentric93 — APPROVED** [review]

**Pull request merged**